lasy/__init__.py

```diff
@@ -1 +1,8 @@
 __version__ = "0.4.0"
+
```

Comment:
This should be discussed: right now, LASY automatically detects whether `cupy` is available and always uses it if it is.